templates/zerver/help/reading-messages-by-stream-or-topic.md

```diff
@@ -3,15 +3,21 @@
 You can view all messages from a specific stream by following the following
 steps.
 
+1. In your view, find the **Streams** section in the left sidebar.
```

Comment:
Remove "In your view, ", since some people consider the "view" to be the message pane.